### DISCUSSION TITLE
Audio is not completely transcribed

### DISCUSSION THREAD

**Discussion by pbussi** [deepgram/community#355] [General help] [answered]
### Which Deepgram product are you using?

Deepgram API

### Details

I invoked the transcribe API with a audio file (duration 17 secs). But only first sentence was transcribed. The second wasnt.

POST /v1/listen?callback=https%3A%2F%2Fnode.theirstory.io%2Ftranscripts%2F651350d723bce5211f183116%2Fdeepgram&diarize=true&model=whisper-large&numerals=false&detect_entities=true

What i am doing wrong?

### If you are making a request to the Deepgram API, what is the full Deepgram URL you are making a request to?

POST /v1/listen?callback=https%3A%2F%2Fnode.theirstory.io%2Ftranscripts%2F651350d723bce5211f183116%2Fdeepgram&diarize=true&model=whisper-large&numerals=false&detect_entities=true

### If you are making a request to the Deepgram API and have a request ID, please paste it below:

2304b538-9f0c-4f7a-b689-c6078c313698

**Comment by pbussi** [comment]
I tried with nova model as well

**Comment by jjmaldonis** [comment]
Hi @pbussi, can you upload the audio file where this is occurring?

If you use the URL below, does the 2nd sentence get transcribed?

`POST /v1/listen?model=whisper-large`

**Comment by pbussi** [comment]
Hi, yes, it fails with whiper-large model also.



Please find attached the original file

**Reply by jjmaldonis** [reply, marked as answer]
Got it, thanks @pbussi. I'm able to reproduce the issue, and it seems to be due to the mp4 file format.

To get the full transcript, you can convert the .mp4 file to a .wav file (or .mp3). Here is the ffmpeg command I used to do the conversion: `ffmpeg -i 271034659-2fb332bd-a343-4400-ab08-5d844c13bd9b.mp4 -ac 2 -f wav 271034659-2fb332bd-a343-4400-ab08-5d844c13bd9b.wav` When you send the .wav file to Deepgram, the full transcript will be returned.

I will pass this issue along to our team to get it fixed.

**Reply by pbussi** [reply]
Thank you for your response! Let me know if you get the issue fixed
